test/tests.js

```diff
@@ -8341,7 +8341,10 @@ describe("Utils", function () {
             set.add(values[1]);
             var index = 0;
             set.forEach(function (value1, value2, passedSet) {
-                assert.strictEqual(value1, value2, "The two value arguments passed to the callback are the same");
+                // HACKHACK: Safari does not pass a duplicate value as the second argument, as the spec suggests.
+                if (value2 !== undefined) {
```

Comment:
It is covered. Safari has `value2 === undefined`. the else would look like
`typescript
else {
  assert.strictEqual(value2, undefined, "Safari")
}
```
which looks a bit silly